### DISCUSSION TITLE
How to get `user.createdAt` when using `jwt`?

### DISCUSSION THREAD

**Discussion by matiastucci** [nextauthjs/next-auth#1768] [Help]
Currently I'm using the default config (not `jwt`) and I'm doing this:

```js
callbacks: {
  session: async (session, user) => {
    session.user.createdAt = user.createdAt
    return Promise.resolve(session)
  },
},
```

but now I want to use [`jwt`](https://next-auth.js.org/configuration/options#jwt) and I'm doing this:

```js
session: async (session, user) => {
  // no session.user.createdAt anymore :(
  return Promise.resolve(session)
},
```
and I don't get the `createdAt` anymore. If I console `session.user` it only has `name`, `email` and `image`. Any idea of how can I obtain the `createdAt` again?

Thanks!

**Comment by balazsorban44** [comment]
https://next-auth.js.org/configuration/callbacks

https://next-auth.js.org/configuration/options
